Source/WebCore/platform/graphics/FontBaseline.h

```diff
@@ -28,7 +28,9 @@
 
 namespace WebCore {
 
-enum FontBaseline { AlphabeticBaseline, IdeographicBaseline };
+enum FontBaseline { AlphabeticBaseline, IdeographicBaseline, CentralBaseline };
+
+enum BaselineSynthesisEdge { Content, Border, Margin };
```

Comment:
I assume these refer to the CSS box which case they should read { ContentBox, BorderBox, MarginBox }

Reply:
Changed